### ISSUE TITLE
Create Telegram chat for SIGTURK

### ISSUE THREAD

**Issue by varie** [sigturk/sigturk#43]

**Comment by varie** [comment]
> Don't we have a standard listed for adherence to open-source and open-protocol tools? If so, we should stick to Matrix.

That's right! The issue was that not many people use it :(. Should we still give it a try?

**Comment by jonorthwash** [comment]
> The issue was that not many people use it

So we have a great opportunity to promote its use!

If someone really wants we could make a bridge between Matrix and another platform, but that seems to defeat the purpose some.

**Comment by mehmetoguzderin** [comment]
IMHO Element (Matrix) UX is actually among the better ones out there, I'd also vouch for its adoption.
